src/Be.Vlaanderen.Basisregisters.Projector/Internal/RetryPolicies/LinearBackOff.cs

```diff
@@ -0,0 +1,55 @@
+namespace Be.Vlaanderen.Basisregisters.Projector.Internal.RetryPolicies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using ConnectedProjections;
+    using Microsoft.Extensions.Logging;
+    using Polly;
+    using SqlStreamStore.Streams;
+
+    internal class LinearBackOff<TException> : MessageHandlingRetryPolicy where TException : Exception
```

Comment:
You can use the Wikipedia style of the word "Backoff": https://en.wikipedia.org/wiki/Exponential_backoff

`LinearBackoff`